src/permitInfo/calculatePermitInfo.ts

```diff
@@ -0,0 +1,154 @@
+import {getTokenPermitInfo, PermitInfo} from 'alfetopito-permit-utils' // import {getTokenPermitInfo, PermitInfo} from '@cowprotocol/permit-utils' // TODO: maybe make the args nicer?
```

Comment:
I don't know WTF is wrong with prettier/eslint/idea which keeps copying the first comment after the imports and appending it after a random import 😖